assets/src/theme/VarPicker.js

```diff
@@ -1,14 +1,36 @@
 import { useEffect, useState } from 'react';
 import { VariableControl } from './VariableControl';
-import { colorToState } from './colorToState';
 import { THEME_ACTIONS, useThemeEditor } from './useThemeEditor';
 import { whileHoverHighlight } from './highlight';
 import { exportCss, exportJson } from './export';
+import { useServerThemes } from './useServerThemes';
 
 export const LOCAL_STORAGE_KEY = 'p4-theme';
 
 const byName = (a, b) => a.name > b.name ? 1 : (a.name === b.name ? 0 : -1);
 
+const diffThemes = (themeA, themeB) => {
+  const added = Object.keys(themeB).filter(k=> !themeA[k]);
```

Comment:
Minor spacing issue here and on the following lines, I think there should be a space before the arrow? 🤔

Reply:
Yeah sometimes I forget to format, since usually the linter catches this. We still have an [open ticket to set up linting for all JS](https://jira.greenpeace.org/browse/PLANET-3641). I applied the changes you suggested, maybe we should pick up the ticket. The changes will probably not be much, I think we just need to decide on the ruleset.